docs/use-cases/observability/clickstack/deployment/hyperdx-clickhouse-cloud.mdx

```diff
@@ -17,11 +17,16 @@ import hyperdx_cloud_landing from '@site/static/images/use-cases/observability/h
 import hyperdx_cloud_datasource from '@site/static/images/use-cases/observability/hyperdx_cloud_datasource.png';
 import hyperdx_create_new_source from '@site/static/images/use-cases/observability/hyperdx_create_new_source.png';
 import hyperdx_create_trace_datasource from '@site/static/images/use-cases/observability/hyperdx_create_trace_datasource.png';
+import {TrackedLink} from '@site/src/components/TrackedLink/TrackedLink.tsx'
 
 <PrivatePreviewBadge/>
 
 This option is designed for users who are using ClickHouse Cloud. In this deployment pattern, both ClickHouse and HyperDX are hosted in ClickHouse Cloud, minimizing the number of components the user needs to self-host.
 
+:::note[New to ClickHouse Cloud?]
+<TrackedLink href="/docs/cloud/overview" eventName="clickstack.deployment.cloud_learn_more.cta">Learn more about ClickHouse Cloud</TrackedLink> or <TrackedLink href="https://clickhouse.cloud/signUp" eventName="clickstack.deployment.cloud_signup.cta" target="_blank" rel="noopener noreferrer">sign up for a free trial</TrackedLink> to get started.
```

Comment:
For galaxy events you need to use only three qualifiers. These map to fields in Galaxy for `namespace.component.eventName` and you can use any arbitrary value for each qualifier.  Better to keep it consistent, especially for namespaces. I'd suggest something like `docs.clickstack.deployment_cloud_learn_more` and `docs.clickstack.deployment_cloud_signup`